app-config.yaml

```diff
@@ -293,6 +293,14 @@ dynatrace:
 # Plugin: Permission
 permission:
   enabled: ${PERMISSION_ENABLED}
+  rbac:
+    admin:
+      users:
+        - name: user:default/alice
+        - name: group:default/admins
+    policies-csv-file: ./rbac/rbac-policy.csv
```

Comment:
Checking with engineering on the best place for this but let's change it to an Environment Variable and that way it can be set on the fly.
So I'd change it to:
`${RBAC_POLICY_FILE}`